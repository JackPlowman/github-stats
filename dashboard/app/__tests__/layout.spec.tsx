import {expect, test} from '@jest/globals';

import {render} from '@testing-library/react';

import Layout from '../layout';

test('renders default layout', async () => {
  // Act
  const {findByText} = render(<Layout children={<p>Hello World</p>} />);
  // Assert
  const navbar = await findByText(/Overview/i);
  expect(navbar).toBeTruthy();
  // check content
  const heading = await findByText(/Hello World/i);
  expect(heading).toBeTruthy();
});
