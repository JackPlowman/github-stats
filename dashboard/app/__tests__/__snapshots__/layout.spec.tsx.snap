// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default layout 1`] = `[Function]`;
